deps/rabbit/src/rabbit.erl

```diff
@@ -1080,6 +1080,7 @@ stop(State) ->
         _  -> rabbit_prelaunch:set_stop_reason(State)
     end,
     rabbit_boot_state:set(stopped),
+    rabbit_db:clear_init_finished(),
```

Comment:
Could you please move this line before `rabbit_boot_state:set(stopped)`?

This is to make sure that we set the boot state as the very last step in the function.